docker-compose.yml.tmpl

```diff
@@ -37,6 +37,7 @@ services:
     restart: always
     volumes:
       - resin-data:/data
+      - /home/${USER}/.asoundrc:/data/.asoundrc
```

Comment:
Looks like you've accidentally placed this in the `anthias-server` section. Please move this to `anthias-viewer`.

Reply:
That would be fine. We can just squash and merge this pull request later if this gets approved